products/esa_s3_ol_2_wfr_nrt.odc-product.yaml

```diff
@@ -0,0 +1,189 @@
+name: s3_ol_2_wfr_nrt
+description: >-
+  Sentinel-3 Level 2 Water Full Resolution (WFR) products in Near Real Time
+  (NRT)
+metadata_type: eo3
+license: CC-BY-4.0
+metadata:
+  product:
+    name: s3_ol_2_wfr_nrt
+storage:
+  crs: 'EPSG:4326'
+  resolution:
+    longitude: 0.003
+    latitude: -0.003
+measurements:
+  - name: A865
+    aliases:
+      - A865
+    units: '1'
+    dtype: uint8
+    nodata: 255
+  - name: A865_err
+    aliases:
+      - A865_err
```

Comment:
At the moment, all aliases are the same as the main measurement names. The aliases are not useful then. Suggest at least make laternative names with different cases, E.g. A865_ERR as an alias to A865_err.